.github/pull_request_template.md

```diff
@@ -28,6 +29,8 @@ Please describe the tests you've added or any tests that already cover this chan
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] I have added any new baselines to the repo
+- [ ] I have assigned reviewers (see [VisIt's PR procedures](https://visit-sphinx-github-user-manual.readthedocs.io/en/develop/dev_manual/pr_create.html#reviewers) for more information).
 
 
-*Don't forget to squash merge when this pull request is approved*
+*Don't forget to squash merge when this pull request is approved.*
+*Wait for CI tests to pass before merging this pull request.*
```

Comment:
If I recall, you were checking into this setting in our repo. I think this should just be turned on, by default, with potentially some way for owners to circumvent if absolutely necessary (maybe).

Reply:
Just activated. Will remove this note.